install_config/topics/ocp_osp_admin_tasks.adoc

```diff
@@ -42,14 +42,21 @@ $ openstack quota set --secgroups 30 --secgroup-rules 200 --ports 200 <project>
 [[osp_accounts_kuryr]]
 ==== Extra steps for Kuryr SDN
 
-If Kuryr SDN is enabled, and particularly if namespace isolation is used,
-increase the next quotas need to be raised: at least 100 security groups,
-500 security group rules, and 500 ports.
-----
-$ openstack quota set --secgroups 100 --secgroup-rules 500 --ports 500 <project>
-----
+If Kuryr SDN is enabled, especially if you are using namespace isolation,
+increase your project's quotas to:
```

Comment:
Can we specify that it's not a global recommendation and you need to consider your needs ?

Reply:
That text was already there. Rest of the documentation can be fixed/extended in different PRs. This is just to adapt it to the kuryr images consumed from OCP repositories instead of OSP ones.